src/Data.jl

```diff
@@ -50,25 +61,35 @@ function readfile(filepath::AbstractString, lineparser::Function)::Problem
     return Problem(examples, basename(filepath))
 end
 
+
+"""
+Writes IO examples to disk by serializing them into a file using HDF5 checking for and appending the `.xio` file ending.
+"""
+function write_IOexamples(filepath::AbstractString, examples::Vector{IOExample})
+    serialize(filepath * (endswith(filepath, ".xio") ? "" : ".xio"), examples)
+end
+
+"""
+Writes IO examples and the corresponding programs to disk by serializing them into a file using HDF5 checking for and appending the `.xiop`.
+"""
+function write_IOPexamples(filepath::AbstractString, examples::Vector{Tuple{Data.IOExample, Any}})
```

Comment:
`Data.IOExample` -> `IOExample`.
This also fixes an error you'd get after merging since the repositories have been renamed.
Also see line 90.